Views/backend/connect/view/main/progress.js

```diff
@@ -0,0 +1,83 @@
+//{namespace name=backend/connect/view/main}
+
+//{block name="backend/connect/view/main/progress"}
+Ext.define('Shopware.apps.Connect.view.main.Progress', {
+
+    extend:'Enlight.app.Window',
+    cls: Ext.baseCSSPrefix + 'connect',
+
+    alias: 'widget.connect-main-migration-progress-window',
+    border: false,
+    autoShow: true,
+    layout: 'anchor',
+    width: 420,
+    height: 80,
+    maximizable: false,
+    minimizable: false,
+    closable: false,
+    footerButton: true,
+    stateful: true,
+    modal: true,
+    inProcess: false,
+
+    /**
+     * Contains the batch size for each request of the generation.
+     */
+    batchSize: 200,
+
+    /**
+     * Contains array with source ids which have to exported
+     */
+    sourceIds: [],
+
+    /**
+     * Contains all snippets for the component
+     * @object
+     */
+    snippets: {
+        title: 'Migration',
```

Comment:
is it ok? Do we need translation for it ?

Reply:
It's the same in german